test/trackerBlocking.js

```diff
@@ -94,6 +127,7 @@ async function rulesetEqual (tds, isRegexSupported, startingRuleId, {
 
     assert.deepEqual(stringifyBlocklist(tds), tdsBefore, 'TDS mutated!')
     if (expectedRuleset) {
+        /** @type {any} */
```

Comment:
Does object work here?
```suggestion
        /** @type {object} */
```

Reply:
I did consider that, but the transform functions have the option to return any type of value, not only Objects. So I think the `{any}` type is accurate in this case.

Reply:
How about 'unknown' this represents all possible types where as any means ignore checking.

Reply:
Hmm, I just tried that but it results in its own `TS2571: Object is of type 'unknown'.` error. Do you mind if I leave this one, I agree it's not ideal but it doesn't seem worth the effort spending to much time on. What do you think?